src/main/java/com/notably/commons/core/path/AbsolutePath.java

```diff
@@ -1,53 +1,127 @@
 package com.notably.commons.core.path;
 
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 
 import com.notably.commons.core.path.exceptions.InvalidPathException;
 
 /**
- * TODO: Add Javadoc
+ * Represents a Path to the block that take reference from the Root node.
```

Comment:
Do you think it would be clearer English to say "Represents the Path to a Block, starting from the Root node."